src/utils.js

```diff
@@ -76,3 +76,22 @@ export async function downloadFile( response ) {
 		}, 100 );
 	}
 }
+
+/*
+ * Trim a string.
+ * @param {string} str The string to trim.
+ * @return {string} The trimmed string.
+ */
+export function trim( str ) {
+	str.trim();
+}
+
+/*
+ * Add quotes to fon name if it ends with a number.
+ * @param {string} familyName The font family name.
+ * @return {string} The font family name with quotes if it ends with a number.
+ */
+
+export function addQuotesToName( familyName ) {
+	return trim( familyName.match( /\d$/ ) ? `'${ familyName }'` : familyName );
```

Comment:
This is only handling font family names that end with a number, so it won't cater to fonts such as [Libre Barcode 128 Text](https://fonts.google.com/specimen/Libre+Barcode+128+Text).

I also noticed that the CSS `font-family` [docs](https://developer.mozilla.org/en-US/docs/Web/CSS/font-family#valid_family_names) mention more than just numbers as a requirement for quotes:

> It is a good practice to quote font family names that contain white space, digits, or punctuation characters other than hyphens.

I'm wondering if it would be better to wrap all font names in quotes regardless of whether the name includes numbers or special characters. Are there any downsides to this? 

I had a look through the [LibFont](https://github.com/Pomax/lib-font/) library to see if there were any helpful functions, I didn't find any but I noticed that they're wrapping font names in quotes on output [here](https://github.com/Pomax/lib-font/blob/master/src/utils/fontface.js#L56).